From 7f2d4051f071678e6bbde03cf04fc834e6fc2f15 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Thu, 4 Jun 2015 18:00:37 +0200
Subject: [PATCH] ARM: dts: bytepanel: Remove unneeded configuration
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Urs Fässler <urs.fassler@bytesatwork.ch>
---
 arch/arm/boot/dts/bytepanel.dtsi | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/arch/arm/boot/dts/bytepanel.dtsi b/arch/arm/boot/dts/bytepanel.dtsi
index 5874191..54aecf4 100644
--- a/arch/arm/boot/dts/bytepanel.dtsi
+++ b/arch/arm/boot/dts/bytepanel.dtsi
@@ -47,19 +47,6 @@
 };
 
 &am33xx_pinmux {
-	pinctrl-names = "default";
-	pinctrl-0 = <&defpins_pins>;
-
-	// ---- TODO implement all systems and remove those pins ----
-	defpins_pins: pinmux_defpins_pins {
-		pinctrl-single,pins = <
-			PIN_SPI0_CS0		0
-			PIN_SPI0_D1		0
-			PIN_UART1_TXD		0
-			PIN_UART1_RXD		0
-		>;
-	};
-
 	// ---- UART0 on X3 ----
 	uart0_pins: pinmux_uart0_pins {
 		pinctrl-single,pins = <
-- 
2.1.4

