From 17ee1ca600b3012ed59cb480d5c01d632e538291 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stephan=20D=C3=BCnner?= <stephan.duenner@bytesatwork.ch>
Date: Thu, 8 Jun 2017 16:39:13 +0200
Subject: [PATCH] spinor: adding m25pe40

---
 drivers/mtd/spi-nor/spi-nor.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/spi-nor/spi-nor.c b/drivers/mtd/spi-nor/spi-nor.c
index f028277fb1ce..428bd22103cb 100644
--- a/drivers/mtd/spi-nor/spi-nor.c
+++ b/drivers/mtd/spi-nor/spi-nor.c
@@ -1190,6 +1190,7 @@ static const struct flash_info spi_nor_ids[] = {
 	{ "m45pe16", INFO(0x204015,  0, 64 * 1024,   32, 0) },
 
 	{ "m25pe20", INFO(0x208012,  0, 64 * 1024,  4,       0) },
+	{ "m25pe40", INFO(0x208013,  0, 64 * 1024,  8,       0) },
 	{ "m25pe80", INFO(0x208014,  0, 64 * 1024, 16,       0) },
 	{ "m25pe16", INFO(0x208015,  0, 64 * 1024, 32, SECT_4K) },
 
-- 
2.11.0

