From 1cd14095e82afb0f8ca4b8282d7af2523c3a6782 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Wed, 25 Jul 2018 18:14:15 +0200
Subject: [PATCH 2/4] sf: ids: Add support for M25PE40 and W25Q128JV

---
 drivers/mtd/spi/spi_flash_ids.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mtd/spi/spi_flash_ids.c b/drivers/mtd/spi/spi_flash_ids.c
index 8132063093..55a831e331 100644
--- a/drivers/mtd/spi/spi_flash_ids.c
+++ b/drivers/mtd/spi/spi_flash_ids.c
@@ -116,6 +116,7 @@ const struct spi_flash_info spi_flash_ids[] = {
 	{"m25p80",	   INFO(0x202014, 0x0, 64 * 1024,    16, 0) },
 	{"m25p16",	   INFO(0x202015, 0x0, 64 * 1024,    32, 0) },
 	{"m25pE16",	   INFO(0x208015, 0x1000, 64 * 1024, 32, 0) },
+	{"m25pE40",	   INFO(0x208013, 0x0,  256 * 256,    8, 0) },
 	{"m25pX16",	   INFO(0x207115, 0x1000, 64 * 1024, 32, RD_QUAD | RD_DUAL) },
 	{"m25p32",	   INFO(0x202016, 0x0,  64 * 1024,    64, 0) },
 	{"m25p64",	   INFO(0x202017, 0x0,  64 * 1024,   128, 0) },
@@ -164,6 +165,7 @@ const struct spi_flash_info spi_flash_ids[] = {
 	{"w25q32bv",	   INFO(0xef4016, 0x0,	64 * 1024,    64, RD_FULL | WR_QPP | SECT_4K) },
 	{"w25q64cv",	   INFO(0xef4017, 0x0,	64 * 1024,   128, RD_FULL | WR_QPP | SECT_4K) },
 	{"w25q128bv",	   INFO(0xef4018, 0x0,	64 * 1024,   256, RD_FULL | WR_QPP | SECT_4K) },
+	{"w25q128jv",	   INFO(0xef7018, 0x0,	64 * 1024,   256, RD_FULL | WR_QPP | SECT_4K) },
 	{"w25q256",	   INFO(0xef4019, 0x0,	64 * 1024,   512, RD_FULL | WR_QPP | SECT_4K) },
 	{"w25q80bw",	   INFO(0xef5014, 0x0,	64 * 1024,    16, RD_FULL | WR_QPP | SECT_4K) },
 	{"w25q16dw",	   INFO(0xef6015, 0x0,	64 * 1024,    32, RD_FULL | WR_QPP | SECT_4K) },
-- 
2.11.0

