From e366b439005fa76256393f95421d714928238aeb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Thu, 4 Jun 2015 15:17:54 +0200
Subject: [PATCH 2/9] am335x_m2: Delete wrong I2C1 configuration
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Urs Fässler <urs.fassler@bytesatwork.ch>
---
 board/bytesatwork/am335x/mux.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/board/bytesatwork/am335x/mux.c b/board/bytesatwork/am335x/mux.c
index c9fbb3c..b9af1f6 100644
--- a/board/bytesatwork/am335x/mux.c
+++ b/board/bytesatwork/am335x/mux.c
@@ -79,14 +79,6 @@ static struct module_pin_mux i2c0_pin_mux[] = {
 	{-1},
 };
 
-static struct module_pin_mux i2c1_pin_mux[] = {
-	{OFFSET(spi0_d1), (MODE(2) | RXACTIVE |
-			PULLUDEN | SLEWCTRL)},	/* I2C_DATA */
-	{OFFSET(spi0_cs0), (MODE(2) | RXACTIVE |
-			PULLUDEN | SLEWCTRL)},	/* I2C_SCLK */
-	{-1},
-};
-
 static struct module_pin_mux gpio0_7_pin_mux[] = {
 	{OFFSET(ecap0_in_pwm0_out), (MODE(7) | PULLUDEN)},	/* GPIO0_7 */
 	{-1},
@@ -148,7 +140,6 @@ void enable_i2c0_pin_mux(void)
 
 void enable_board_pin_mux(void)
 {
-	configure_module_pin_mux(i2c1_pin_mux);
 	configure_module_pin_mux(gpio0_7_pin_mux);
 	configure_module_pin_mux(mii1_pin_mux);
 	configure_module_pin_mux(mmc0_pin_mux);
-- 
2.1.4

