From 14f978572bea6fccd534701baa9152cf1cf546b7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stephan=20D=C3=BCnner?= <stephan.duenner@bytesatwork.ch>
Date: Thu, 31 Mar 2016 14:35:12 +0200
Subject: [PATCH 7/8] devicetree: adding muxing and enable UART1

Signed-off-by: Daniel Ammann <daniel.ammann@bytesatwork.ch>
---
 arch/arm/boot/dts/bytepanel.dtsi | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/arch/arm/boot/dts/bytepanel.dtsi b/arch/arm/boot/dts/bytepanel.dtsi
index 0526656..a171ce1 100644
--- a/arch/arm/boot/dts/bytepanel.dtsi
+++ b/arch/arm/boot/dts/bytepanel.dtsi
@@ -133,13 +133,32 @@
 			PIN_UART0_RTSN		(MUX_MODE3 | PIN_INPUT_PULLUP | SLEWCTRL_FAST)	// I2C1_SCL
 		>;
 	};
+
+	// ---- UART1 on X6  ----
+	uart1_pins: pinmux_uart1_pins {
+		pinctrl-single,pins = <
+			PIN_UART1_RTSN		(MUX_MODE0 | PIN_OUTPUT)		// UART1_RTSn
+			PIN_UART1_CTSN		(MUX_MODE0 | PIN_INPUT)			// UART1_CTSn
+			PIN_UART1_TXD		(MUX_MODE0 | PIN_OUTPUT)		// UART1_TXD
+			PIN_UART1_RXD		(MUX_MODE0 | PIN_INPUT)			// UART1_RXD
+			PIN_MCASP0_FSR		(MUX_MODE7 | PIN_OUTPUT_PULLDOWN)	// GPIOMUX1
+			PIN_MCASP0_ACLKX	(MUX_MODE7 | PIN_OUTPUT_PULLDOWN)	// GPIOMUX2
+			PIN_MCASP0_ACLKR	(MUX_MODE7 | PIN_OUTPUT_PULLDOWN)	// GPIOMUX3
+		>;
+	};
 };
 
 // ---- UART0 on X3 ----
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_pins>;
+	status = "okay";
+};
 
+// ---- UART1 on X6 ----
+&uart1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&uart1_pins>;
 	status = "okay";
 };
 
-- 
2.1.4

