From ec10a789a637dba6a32efbaaa1b62fcd7c58e332 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stephan=20D=C3=BCnner?= <stephan.duenner@bytesatwork.ch>
Date: Thu, 8 Jun 2017 16:39:13 +0200
Subject: [PATCH 16/17] spinor: adding m25pe40

---
 drivers/mtd/spi-nor/spi-nor.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/spi-nor/spi-nor.c b/drivers/mtd/spi-nor/spi-nor.c
index 37e4135..e84918c 100644
--- a/drivers/mtd/spi-nor/spi-nor.c
+++ b/drivers/mtd/spi-nor/spi-nor.c
@@ -810,6 +810,7 @@ static const struct flash_info spi_nor_ids[] = {
 	{ "m45pe16", INFO(0x204015,  0, 64 * 1024,   32, 0) },
 
 	{ "m25pe20", INFO(0x208012,  0, 64 * 1024,  4,       0) },
+	{ "m25pe40", INFO(0x208013,  0, 64 * 1024,  8,       0) },
 	{ "m25pe80", INFO(0x208014,  0, 64 * 1024, 16,       0) },
 	{ "m25pe16", INFO(0x208015,  0, 64 * 1024, 32, SECT_4K) },
 
-- 
2.1.4

