From 6824be1d8bca952d6b10fd18613e2eafdb8f6bb0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stephan=20D=C3=BCnner?= <stephan.duenner@bytesatwork.ch>
Date: Thu, 8 Jun 2017 16:39:13 +0200
Subject: [PATCH 16/25] spinor: adding m25pe40

---
 drivers/mtd/spi-nor/spi-nor.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/spi-nor/spi-nor.c b/drivers/mtd/spi-nor/spi-nor.c
index 21dde5249085..6feded7ae69b 100644
--- a/drivers/mtd/spi-nor/spi-nor.c
+++ b/drivers/mtd/spi-nor/spi-nor.c
@@ -962,6 +962,7 @@ static const struct flash_info spi_nor_ids[] = {
 	{ "m45pe16", INFO(0x204015,  0, 64 * 1024,   32, 0) },
 
 	{ "m25pe20", INFO(0x208012,  0, 64 * 1024,  4,       0) },
+	{ "m25pe40", INFO(0x208013,  0, 64 * 1024,  8,       0) },
 	{ "m25pe80", INFO(0x208014,  0, 64 * 1024, 16,       0) },
 	{ "m25pe16", INFO(0x208015,  0, 64 * 1024, 32, SECT_4K) },
 
-- 
2.11.0

