From 18bbba5b11edd49ad7bf33def359ec7319357c44 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stephan=20D=C3=BCnner?= <stephan.duenner@bytesatwork.ch>
Date: Thu, 15 Jun 2017 12:55:34 +0200
Subject: [PATCH] EMIF: Fix lcd pixel shift

---
 board/bytesatwork/am335x/ram.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/bytesatwork/am335x/ram.c b/board/bytesatwork/am335x/ram.c
index 6da3a60..4a811d5 100644
--- a/board/bytesatwork/am335x/ram.c
+++ b/board/bytesatwork/am335x/ram.c
@@ -285,6 +285,7 @@ void sdram_init(void)
 	/* TODO: Initialize UARTS earlier again so that we can print the configuration */
 	//printf("use ram configuration %u (%s)\n", ram->config, m2config_get_ram_name(ram->config));
 	ram->init();
+	writel(0x00ffff38, EMIF1_BASE + 0x54 );
 }
 
 const struct dpll_params *get_dpll_ddr_params(void)
-- 
2.1.4

