From aedfdf79c9ae3735e77fda7c641faa45ad90b1c0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Tue, 8 Oct 2013 10:38:54 +0200
Subject: [PATCH 2/3] M2: Enable u-boot environment in NAND

---
 include/configs/am335x_m2.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/configs/am335x_m2.h b/include/configs/am335x_m2.h
index d905d67..c50aea9 100644
--- a/include/configs/am335x_m2.h
+++ b/include/configs/am335x_m2.h
@@ -268,7 +268,9 @@
 #define CONFIG_ENV_OVERWRITE		1
 #define CONFIG_SYS_CONSOLE_INFO_QUIET
 
-#define CONFIG_ENV_IS_NOWHERE
+#define CONFIG_ENV_IS_IN_NAND
+#define CONFIG_ENV_OFFSET              0x260000 /* environment starts here */
+#define CONFIG_SYS_ENV_SECT_SIZE       (128 << 10)     /* 128 KiB */
 
 /* Defines for SPL */
 #define CONFIG_SPL
-- 
1.7.10.4

