From 82d82913aaf8faa9105f3f5b9a82175c2fc8406f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Fri, 21 Feb 2014 13:29:05 +0100
Subject: [PATCH 3/3] M2: Disable cursor on framebuffer

---
 include/configs/am335x_m2.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/am335x_m2.h b/include/configs/am335x_m2.h
index c50aea9..14ee867 100644
--- a/include/configs/am335x_m2.h
+++ b/include/configs/am335x_m2.h
@@ -77,7 +77,8 @@
 	"nand_rootfs_size=0x8200000\0" \
 	"mmc_load_uimage=fatload mmc ${mmc_dev} ${loadaddr} ${bootfile}\0" \
 	"bootargs_defaults=setenv bootargs " \
-		"console=${console} \0" \
+		"console=${console} " \
+		"vt.global_cursor_default=0 \0" \
 	"mmc_args=run bootargs_defaults;" \
 		"setenv bootargs ${bootargs} " \
 		"root=${mmc_root} " \
-- 
1.7.10.4

